manifests/database.pp

```diff
@@ -23,7 +23,7 @@
     logoutput   => 'on_failure',
     user        => 'apache',
     refreshonly => true,
-    require     => File['/etc/pulp/server.conf'],
```

Comment:
I think this is still need it because the migration should run after all needed information is populated.